core/plugins/org.polarsys.capella.core.sirius.analysis/src/org/polarsys/capella/core/sirius/analysis/delete/CapellaDeleteActionHook.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2006, 2017 THALES GLOBAL SERVICES.
+ * Copyright (c) 2006, 2017, 2020 THALES GLOBAL SERVICES.
```

Comment:
It shall be "2006, 2020"
But don't bother with updating them manually.

Reply:
I will wait for the build to be done and if successful I will fix it in another commit and we can merge right after.